src/repository/WebhookFileRepository.js

```diff
@@ -32,11 +32,33 @@ class WebhookFileRepository {
   deleteById = async (id: string): Promise<void> =>
     this._fileManager.deleteFile(`${id}.json`);
 
-  getAll = async (): Promise<Array<Webhook>> =>
-    this._fileManager.getAllData();
+  getAll = async (userID: ?string = null): Promise<Array<Webhook>> => {
+    const allData = this._fileManager.getAllData();
 
-  getById = async (id: string): Promise<?Webhook> =>
-    this._fileManager.getFile(`${id}.json`);
+    if (userID) {
+      return Promise.resolve(
```

Comment:
I don't think that you need to return a promise here.  There should be a way to just return objects since you already have `async` on the function.